src/index.jsx

```diff
@@ -12,21 +12,15 @@ import {
   faTriangleExclamation,
   faUser,
 } from '@fortawesome/free-solid-svg-icons';
-import ReactGA from 'react-ga';
 import { Provider } from 'react-redux';
 import { BrowserRouter } from 'react-router-dom';
 import store from './redux/store';
-import { gaTracking } from './params';
 import { basename } from './localconf';
 import App from './App';
 import '@fontsource/raleway';
 import './base.css';
 import './icon.css';
 
-// Google Analytics
-ReactGA.initialize(gaTracking);
-ReactGA.pageview(window.location.pathname + window.location.search);
-
```

Comment:
@skyzhangty does this need to be replaced with ga4 or it is superfluous and removed on purpose?

Reply:
This is removed on purpose. We have a ```Layout``` component that's the first component in router and is always going to the rendered. In this ```Layout``` component, we set up Google Analytics. I just think there's no need to set it up again in the index.jsx again. I tested it even when I go to a page directly (not via index page) such as the explorer page, the GA related scripts are added to the HTML.